onnxruntime/core/providers/js/operators/instance_norm.h

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/providers/js/js_kernel.h"
+
+namespace onnxruntime {
+namespace js {
+
+class InstanceNorm : public JsKernel {
+ public:
+  InstanceNorm(const OpKernelInfo& info) : JsKernel(info) {
+    ORT_ENFORCE(info.GetAttr<float>("epsilon", &epsilon_).IsOK());
```

Comment:
Use GetAttrOrDefault instead of GetAttr which requires that the attribute is set.